src/sidebar/components/HypothesisApp.js

```diff
@@ -162,6 +165,8 @@ function HypothesisApp({ auth, bridge, settings, session, toastMessenger }) {
     session.logout();
   };
 
+  // eslint-disable-next-line no-unused-vars
+  const { status, ...authHelpPanel } = authState;
```

Comment:
It wasn't clear to me at first why this was done. I see that if trying to pass `authState` directly to `HelpPanel` you get an error, even though the types are compatible, due to the way TS's compatibility heuristics work. After a little more experimentation I found this also works, and I think is more obvious:

```diff
-        <HelpPanel auth={authHelpPanel} />
+        <HelpPanel auth={authState.status === 'logged-in' ? authState : {}} />
```

Reply:
👍🏼